============================= test session starts ==============================
platform linux -- Python 3.8.10, pytest-7.1.2, pluggy-1.0.0
rootdir: /home/runner/work/coverrage_badge/coverrage_badge
plugins: cov-3.0.0
collected 0 items / 1 error

==================================== ERRORS ====================================
_________________ ERROR collecting apps/tests/test_program.py __________________
ImportError while importing test module '/home/runner/work/coverrage_badge/coverrage_badge/apps/tests/test_program.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
apps/tests/test_program.py:4: in <module>
    import mock
E   ModuleNotFoundError: No module named 'mock'
- generated xml file: /home/runner/work/coverrage_badge/coverrage_badge/pytest_cov.xml -

---------- coverage: platform linux, python 3.8.10-final-0 -----------
Name                         Stmts   Miss  Cover   Missing
----------------------------------------------------------
apps/program.py                 12      8    33%   5-9, 12-13, 16-17
apps/tests/test_program.py      12      8    33%   6-19
apps/utils.py                   10      7    30%   2-5, 8-11, 14
----------------------------------------------------------
TOTAL                           34     23    32%

2 files skipped due to complete coverage.

=========================== short test summary info ============================
ERROR apps/tests/test_program.py
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
=============================== 1 error in 0.26s ===============================
